lucene/memory/src/test/org/apache/lucene/index/memory/TestMemoryIndex.java

```diff
@@ -618,4 +622,21 @@ public void testToStringDebug() {
         "fields=2, terms=2, positions=3", mi.toStringDebug());
   }
 
+  public void testExplain() throws ParseException, IOException {
```

Comment:
I don't think this is specific to MemoryIndex, so can you move this test to TestSpansExplanations?

Reply:
Done